src/pages/index.svelte

```diff
@@ -77,6 +90,21 @@
 		</div>
 	</Card>
 
+	<Dialog
+		bind:this={dialog}
+		aria-labelledby="dialog-title"
+		aria-describedby="dialog-content">
+		<Title id="dialog-title">Message to {$currPlayer.name}</Title>
+		<Content id="dialog-content">
+			{dialogMessage}
+		</Content>
+		<Actions>
+			<Button>
+			<Label>Ok</Label>
+			</Button>
+		</Actions>
+	</Dialog>
```

Comment:
Left out handlers, off button and anything else deemed unnecessary.

Reply:
good idea!